bot/handlers/commands.py

```diff
@@ -5,43 +5,10 @@
 import bot.keyboards as kb
 from bot.middlewares.i18n import _
 from bot.misc import dp
+from bot.db import User
 
 
-@dp.message_handler(text_startswith='/start ')
-async def deep_link(msg: types.Message):
-    deep_link = msg.text[7:]
-    await msg.answer(_("Deep link {}").format(deep_link))
-
-
-@dp.message_handler(commands=['start'])
-async def start(msg: types.Message):
-    await msg.answer(_("Hello! I'm Asyncbot."))
-
-    
-@dp.message_handler(commands=['settings'])
-async def settings(msg: types.Message):
-    await msg.answer(
-        _("Settings"), reply_markup=kb.settings())
-
-
-@dp.message_handler(commands=['help'])
-async def help(msg: types.Message):
-    await msg.answer(_("Help"))
-
-
-@dp.message_handler(commands=['privacy'])
-async def privacy(msg: types.Message):
-    await msg.answer(_("Privacy"))
-
-
-@dp.message_handler(commands=['id'])
-async def getid(msg: types.Message):
-    await msg.answer(_("User id: {uid}\nChat id: {cid}").format(
-        uid=hcode(msg.from_user.id),
-        cid=hcode(msg.chat.id)))
-
-
-@dp.message_handler(commands=['cancel'], state='*')
-async def cancel(msg: types.Message, state: FSMContext):
-    await state.finish()
-    await msg.answer(_("Canceled"), reply_markup=types.ReplyKeyboardRemove())
+@dp.message_handler(commands=["card"])
+async def user_card(msg: types.Message):
+    user = User.get_user(types.User.get_current())
```

Comment:
can you explain pls how it works ? what is types.User.get_current() ? 
what returned ?
we can write like that? 
create method here
```
def params(tg_user):
  {
    user_id=tg_user.id,
    username=tg_user.username,
    first_name=tg_user.first_name,
    flast_name=tg_user.last_name,
    locale=tg_user.locale,
  }
``` 
and write
```
user = User.get_or_create(params)
```